backend/src/main/java/com/redhat/cloud/notifications/routers/internal/InternalResource.java

```diff
@@ -436,4 +443,30 @@ public Response unlinkDefaultBehaviorToEventType(@Context SecurityContext sec, @
             return Response.status(BAD_REQUEST).build();
         }
     }
+
+    @PUT
+    @Path("/daily-digest/time-preference/{orgId}")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    @Transactional
+    @RolesAllowed(ConsoleIdentityProvider.RBAC_INTERNAL_ADMIN)
+    public Response saveDailyDigestTimePreference(@NotNull LocalTime expectedTime, @RestPath("orgId") String orgId) {
```

Comment:
Nitpick: `@RestPath` can automagically map the method arg to the path param, the name can be omitted.
```suggestion
    public Response saveDailyDigestTimePreference(@NotNull LocalTime expectedTime, @RestPath String orgId) {
```